package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-filings-ui",
-  "version": "5.3.5",
+  "version": "5.3.6",
```

Comment:
Package lock needs to be updated, after the packages are created for `bcrs-shared-components`

Reply:
Outdated conversation. Resolving it.